lib/rucio/tests/test_pfns.py

```diff
@@ -0,0 +1,70 @@
+# Copyright European Organization for Nuclear Research (CERN)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Authors:
+# - Mario Lassnig, <mario.lassnig@cern.ch>, 2017
+
+from nose.tools import assert_equal
+
+from rucio.rse import rsemanager as rsemgr
+
+
+class TestPFNs(object):
+
+    def test_pfn_srm(self):
+        """ PFN (CORE): Test the splitting of PFNs with SRM"""
+
+        rse_info = rsemgr.get_rse_info('MOCK')
+        proto = rsemgr.create_protocol(rse_info, 'read', scheme='srm')
+        pfn = 'srm://mock.com:8443/rucio/tmpdisk/rucio_tests/whatever'
```

Comment:
`pfns = [''srm://mock.com:8443/rucio/tmpdisk/rucio_tests/whatever'',
               ''srm://mock.com:8443/srm/managerv2?SFN=/rucio/tmpdisk/rucio_tests/whatever''
                ''srm://mock.com:8443/srm/v2/server?SFN=/rucio/tmpdisk/rucio_tests/whatever'']
 for pfn in pfns:
    ret = proto...`

Reply:
well, the asserts are different for all cases

Reply:
I have maybe overlooked the test but normally the asserts should stay the same

Reply:
they have different ports, hostnames, prefixes, etc across the protocols

Reply:
? 
port(8443), hostname(mock.com) and prefixes(/rucio/tmpdisk/rucio_tests/) are the same for the three urls, no ?

Reply:
right, my eyesight is bad